Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace.package]
-version = "2.7.0"                               # managed by cargo-workspaces, see below
+version = "2.7.1-rc.1"                               # managed by cargo-workspaces, see below
```

Comment:
why rc.1? Is it because we're going to testnet first?

Reply:
yes